internal/surveyext/validators.go

```diff
@@ -0,0 +1,70 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package surveyext
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+
+	"github.com/Masterminds/semver"
+	"github.com/pkg/errors"
+)
+
+var (
+	githubOwnerRegexp = regexp.MustCompile("^(([a-zA-Z0-9-]+)|([a-zA-Z0-9-]+\\/[a-zA-Z0-9-]+))$")
```

Comment:
Short term: it would be nice to copy this regex into https://github.com/elastic/package-spec/blob/92ee6e42f2e89456ae4ef5f66531abbc4a23ead4/versions/1/manifest.spec.yml#L270-L272.

Longer term: it would nice to expose this regex via a golang API from the package-spec. That way the package-spec can be the single source of truth for this regex.

Reply:
https://github.com/elastic/package-spec/issues/163

Reply:
https://github.com/elastic/package-spec/issues/163